integrations/kube-agent-updater/go.mod

```diff
@@ -1,22 +1,24 @@
 module github.com/gravitational/teleport/integrations/kube-agent-updater
 
-go 1.19
+go 1.21
+
+toolchain go1.21.1
```

Comment:
v13 is still running on Go 1.20, this won't work.

Reply:
Yes, this was accidental, fixed now